RELEASE/scripts/autoscend/iotms/mr2022.ash

```diff
@@ -232,6 +232,16 @@ void prioritizeGoose() //prioritize Goose only if we still have things to get
 	set_property("auto_prioritizeGoose", false);
 }
 
+boolean auto_haveMaydayContract()
+{
+	if (get_property("hasMaydayContract").to_boolean() && auto_is_valid($item[gaffer's tape]))
```

Comment:
Thoughts on checking `MayDay™ supply package` instead of the tape? Figure can't even use tape or survival knife if we can't open the package. I'll approve as is though as this is minor, up to you if ya wanna change

Reply:
I'm not 100% sure that Mr Store packages are actually restricted by Standard as they simply bind an IotM to your account.
I think it's probably fine to leave like this & we'll find out when Mayday goes out of Standard. We haven't had an IotM in this situation for quite a while and there are always things missed when Standard rotates anyway that we end up having to make changes anyway.